### ISSUE TITLE
Tabulator Pagination icons still don't work on 6.5.1 on Safari

### ISSUE THREAD

**Issue by PedroCavaleiro** [rjaros/kvision#481]
After updating KVision to 6.5.1 I noticed that the tabulator pagination icons are still not working as it was expected on Safari the caches were cleared before opening the issue

Browser: Safari 16.5
OS: macOS Ventura 13.4
KVision: 6.5.1
Kotlin: 1.8.22

![image](https://github.com/rjaros/kvision/assets/1993871/f1ab4bd4-4331-46dc-9c27-db588dcf592f)

**Comment by PedroCavaleiro** [comment]
After discussing with @rjaros 

It was suggested to use the following workaround 

```kotlin
langs = obj {
    default = obj {
        pagination = obj {
            first = "<<"
            last = ">>"
            prev = "<"
            next = ">"
        }
    }
}
```

**Comment by rjaros** [comment]
Fixed in 6.5.2
